src/domain/entities/CustomError.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  TransactionRouting,
+  transactionRoutingDefault,
+} from "./TransactionRouting";
+
+class CustomError {
+  private field: string;
+  private message: string;
+
+  constructor(field: string, message: string) {
+    this.field = field;
+    this.message = message;
+  }
+
+  getField(): string {
+    return this.field;
+  }
+
+  getMessage(): string {
+    return this.message;
```

Comment:
This is an error message coming from the API

Reply:
👍 it may still require translation but we've not yet decided how to handle that (I think there is a suggestion in the ch-node-utils lib README around this).